src/legacy/ui/public/courier/fetch/components/_shard_failure_modal.scss

```
@@ -0,0 +1,47 @@
+// set width and height to fixed values to prevent resizing when you switch tabs
+.shardFailureModal {
+     min-height: 75vh;
+     width: 768px;
+}
+
+// displaying a EuiCodeBlock inside a tab inside a modal, you've to set the width 
+// to prevent overlapping the modal when the content contains very long lines
+.shardFailureModal__codeblock {
+    width: 700px;
+}
+
+.shardFailure_descHeader{
+    margin-bottom: $euiSizeM;
+}
+
+.shardFailureModal__descTitle {
+    width: 20%!important;
+    margin-top: $euiSizeS;
+}
+
+.shardFailureModal__descValue {
+    width: 80%!important;
```

You need a space before the `!`
```suggestion
    width: 80% !important;
```

---

done!